tools/expotools/src/commands/AndroidUnitTests.ts

```diff
@@ -0,0 +1,57 @@
+import spawnAsync from '@expo/spawn-async';
+import chalk from 'chalk';
+
+import * as Directories from '../Directories';
+import * as Packages from '../Packages';
+
+const ANDROID_DIR = Directories.getAndroidDir();
+
+const excludedInTests = [
+  'expo-module-template',
+  'expo-bluetooth',
+  'expo-notifications',
+  'expo-in-app-purchases',
+  'expo-splash-screen',
+  'expo-updates'
```

Comment:
```suggestion
  'expo-updates',
  'expo-splash-screen'
```

Reply:
I believe we should keep these excluded unimodules in one place across whole  repo because it's already n'th place for the same list 😉

Reply:
There already is `expo-splash-screen` in this list ;)

I couldn't agree more. However, that is not easy and straightforward task. First of all, those list does not necessarily have to be identical. If we mean to unify them, this need to be thought thru, I am afraid.

What I am hoping at the very least is that whenever we forget to exclude unimodule from any list it should be excluded, CI should fail. It is not best Developer Experience, but good first step.